gtests/net/mptcp/regressions/disconnect.pkt

```diff
@@ -0,0 +1,13 @@
+// This test case covers the regression fixed by ("mptcp: be careful on MPTCP-level ack.")
+// bad read on on unconnected socket
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
+0.0   socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0.0  fcntl(3, F_GETFL) = 0x2 (flags O_RDWR)
++0.0  fcntl(3, F_SETFL, O_RDWR|O_NONBLOCK) = 0
+
++0.0 connect(3, ..., ...) = -1 EINPROGRESS (Operation now in progress)
++0.0   > S 0:0(0) <mss 1460,sackOK,TS val 100 ecr 0,nop,wscale 8,mpcapable v1 flags[flag_h] nokey>
+
++0.0 shutdown(3, SHUT_WR) = 0
```

Comment:
should we maybe rename this test not to have it too generic? e.g. `unconnected_shutdown.pkt` (or `connect_in_progress_shutdown.pkt` or something else :) )